brave/brave-core
# feat(wallet): Portfolio Graph Layout Updated

## Description 
Updates the `Portfolio` Graph Layout to match the new designs in Figma
- Adds a new `Line Chart Controls` menu
- Adds `Fiat` and `Percentage` change sections
- Persists the users `Time Line History` selection


Resolves <https://github.com/brave/brave-browser/issues/29996>



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Go to the `Portfolio` tab.
2. Test selecting different `Time Lines`.
3. Test `Fiat` and `Percentage` changes are accurate.
4. Refresh the page and make sure that your selected `Time Line` is persisted.

https://user-images.githubusercontent.com/40611140/234662166-3fe3ceb5-9874-4cfc-8b3a-8d58e3142dd4.mov
